### PR TITLE
feat: Added haptic feedback for receive and send screen

### PR DESCRIPTION
**Pull request by Aditi-Singh16**
Fixes issue #1051

### CODE REVIEW

**Comment by nicolasburtey** [comment]
@UncleSamtoshi does it cover all the payment success/failure flow?

**Review by Extheoisah — COMMENTED** [review]
tested the code and this works fine on physical devices for receive and send.
Nice @Aditi-Singh16

**Comment by Extheoisah** [comment]
> @UncleSamtoshi does it cover all the payment success/failure flow?

It doesn't work for payment failure. But it should be fairly easy to add.

File: app/screens/receive-bitcoin-screen/receive-btc.tsx [hunk 1]
```diff
@@ -448,6 +449,12 @@ const ReceiveBtc = () => {
     )
   }
 
+  if (state === PaymentRequestState.Paid) {
```
Comment (Extheoisah) [inline app/screens/receive-bitcoin-screen/receive-btc.tsx:452]: we can also check for payment failure.
```suggestion
  if (state === PaymentRequestState.Paid || PaymentRequestState.Error) {
```

Comment (Aditi-Singh16) [reply app/screens/receive-bitcoin-screen/receive-btc.tsx:452]: Okay! added

File: app/screens/receive-bitcoin-screen/receive-btc.tsx [hunk 2]
```diff
@@ -448,6 +449,16 @@ const ReceiveBtc = () => {
...
+    ReactNativeHapticFeedback.trigger("notificationSuccess", {
+      ignoreAndroidSystemSettings: true,
+    })
+  } else if (state === PaymentRequestState.Error) {
+    ReactNativeHapticFeedback.trigger("notificationSuccess", {
```
Comment (nicolasburtey) [inline app/screens/receive-bitcoin-screen/receive-btc.tsx:457]: shouldn't this be `notificationError`?

File: app/screens/receive-bitcoin-screen/receive-usd.tsx [hunk 3]
```diff
@@ -364,6 +365,16 @@ const ReceiveUsd = () => {
     errorMessage = LL.ReceiveWrapperScreen.error()
   }
 
+  if (state === PaymentRequestState.Paid) {
+    ReactNativeHapticFeedback.trigger("notificationSuccess", {
+      ignoreAndroidSystemSettings: true,
+    })
+  } else if (state === PaymentRequestState.Error) {
+    ReactNativeHapticFeedback.trigger("notificationSuccess", {
```
Comment (nicolasburtey) [inline app/screens/receive-bitcoin-screen/receive-usd.tsx:373]: shouldn't this be notificationError?

Comment (Extheoisah) [reply app/screens/receive-bitcoin-screen/receive-usd.tsx:373]: Yeah should be notificationError.

Comment (Aditi-Singh16) [reply app/screens/receive-bitcoin-screen/receive-usd.tsx]: Sorry! updated

**Comment by Extheoisah** [comment]
@Aditi-Singh16 Please could you perhaps add it to the stablesats conversion screen also here https://github.com/GaloyMoney/galoy-mobile/blob/a8789ecfce436f67d00cee649b98b86776be16fc/app/screens/conversion-flow/conversion-confirmation-screen.tsx#L138

**Comment by Aditi-Singh16** [comment]
> @Aditi-Singh16 Please could you perhaps add it to the stablesats conversion screen also here
> 
> https://github.com/GaloyMoney/galoy-mobile/blob/a8789ecfce436f67d00cee649b98b86776be16fc/app/screens/conversion-flow/conversion-confirmation-screen.tsx#L138
> 
> 
> and here incase of error
> https://github.com/GaloyMoney/galoy-mobile/blob/a8789ecfce436f67d00cee649b98b86776be16fc/app/screens/conversion-flow/conversion-confirmation-screen.tsx#L154

Yes, done!

**Review by Extheoisah — APPROVED** [review]
Looks good to me!

**Pull request merged**